### PR TITLE
Vdb 1669 add eth b prod

### PR DESCRIPTION
**Pull request by elizabethengelman**
ETH-B was added to mcdTransformer.toml using the code generator defined in #344 in commit c7c9a61. It currently still needs some manual tweaking to make sure everything works as expected.

### CODE REVIEW

File: environments/extractDiffs.toml [hunk 1]
```diff
@@ -203,6 +235,14 @@
...
+        deployed = 10933898
```
Comment (rmulhol) [inline environments/extractDiffs.toml]: Do we need the OSM contracts in this config?

File: environments/mcdTransformers.toml [hunk 2]
```diff
@@ -1,1667 +1,1667 @@
...
+    "vat_rely",
+    "vat_slip",
+    "vat_suck",
+    "vow_fess",
+    "vow_file_auction_attributes",
+    "vow_file_auction_address",
+    "vow_flog",
+    "vow_heal",
+    "yank",
+    "flip_eth_b_v1_1_3"
```
Comment (rmulhol) [inline environments/mcdTransformers.toml]: ndb but maybe it'd be cool to put this one close to the other flip storage transformers? Looks like we might have to accept some disorder if we're adding these things programmatically, but wondering if it'd be beneficial to generally clean up after ourselves for ease of manual parsing

File: environments/mcdTransformers.toml [hunk 3]
```diff
@@ -1,1667 +1,1667 @@
...
+        "MCD_FLIP_USDT_A_1_1_1",
+        "MCD_FLIP_WBTC_A_1_0_6",
+        "MCD_FLIP_WBTC_A_1_0_9",
+        "MCD_FLIP_WBTC_A_1_1_0",
+        "MCD_FLIP_ZRX_A_1_0_8",
+        "MCD_FLIP_ZRX_A_1_0_9",
+        "MCD_FLIP_ZRX_A_1_1_0",
+        "MCD_FLOP_1_0_1",
+        "MCD_FLOP_1_0_9",
+        "MCD_FLIP_ETH_B_1_1_3"
```
Comment (rmulhol) [inline environments/mcdTransformers.toml]: I guess the same thing is true here, so maybe not worth it if it's going to create a ton of work to maintain

File: plugins/execute/transformerExporter.go [hunk 4]
```diff
@@ -132,126 +133,127 @@ var Exporter exporter
...
+        vat_grab.EventTransformerInitializer,
+        median_diss_single.EventTransformerInitializer,
+        pot_cage.EventTransformerInitializer,
+        cat_file_vow.EventTransformerInitializer,
+        vat_flux.EventTransformerInitializer,
+        vat_rely.EventTransformerInitializer,
+        vat_move.EventTransformerInitializer,
+        spot_file_pip.EventTransformerInitializer,
+        vat_file_ilk.EventTransformerInitializer,
+        osm_change.EventTransformerInitializer,
```
Comment (rmulhol) [inline plugins/execute/transformerExporter.go]: could be cool to alphabetize and format this file, if only to make the diff clearer (added/removed transformers)

File: test_config/test_config.go [hunk 5]
```diff
@@ -48,15 +48,17 @@ func SetTestConfig() {
 	viper.AddConfigPath("$GOPATH/src/github.com/makerdao/vdb-mcd-transformers/environments/")
 
 	viper.SetConfigName("testDatabase")
-	mergeConfigErr := viper.ReadInConfig()
-	if mergeConfigErr != nil {
-		log.Fatal(mergeConfigErr)
+	readConfigerr := viper.ReadInConfig()
```
Comment (rmulhol) [inline test_config/test_config.go]: err => Err

File: transformers/test_data/config_values.go [hunk 6]
```diff
@@ -43,20 +43,20 @@ func FlipV100Addresses() []string {
...
-		"MCD_FLIP_TUSD_A_1.1.0",
-		"MCD_FLIP_USDC_A_1.1.0",
-		"MCD_FLIP_USDC_B_1.1.0",
-		"MCD_FLIP_USDT_A_1.1.1",
-		"MCD_FLIP_WBTC_A_1.1.0",
-		"MCD_FLIP_ZRX_A_1.1.0",
+		"MCD_FLIP_BAT_A_1_1_0",
+		"MCD_FLIP_COMP_A_1_1_2",
+		"MCD_FLIP_ETH_A_1_1_0",
+		"MCD_FLIP_LINK_A_1_1_2",
```
Comment (rmulhol) [inline transformers/test_data/config_values.go]: need to add ETH_B here?

**Review by rmulhol — APPROVED** [review]

File: environments/extractDiffs.toml (hunk 1 above)
Comment (elizabethengelman) [reply environments/extractDiffs.toml]: nope, probably not. good call!

**Pull request merged**